### PR TITLE
Endoc 334 add language for storage class requirements 

### PR DESCRIPTION
**Pull request by joewhite101**

### CODE REVIEW

File: vuepress/docs/v6.3.2/docs/reference/caching-and-clustering.md [hunk 1]
```diff
@@ -12,6 +12,28 @@ The clustering of microservices built to add functionality to an Entando applica
 have a separate clustering configuration and setup depending on the implementation and choices made in creating those microservices. See the documentation
 for your microservices caching implementation for details on configuring and deploying clustered microservices.
 
+## Storage Requirements for Clustered Entando Apps
+
+In order to scale an Entando Application across multiple nodes you must provide a storage class that supports
+a `ReadWriteMany` access policy. There are many ways to accomplish this including using dedicated storage providers
+like GlusterFS or others. he cloud Kubernetes providers also provide clustered storage options specific to their implementation like Google Cloud File in GKE or Azure Files in AKS.
```
Comment (avdev4j) [inline vuepress/docs/v6.3.2/docs/reference/caching-and-clustering.md:19]:
```suggestion
like GlusterFS or others. The cloud Kubernetes providers also provide clustered storage options specific to their implementation like Google Cloud File in GKE or Azure Files in AKS.
```

File: vuepress/docs/v6.3.2/docs/reference/caching-and-clustering.md [hunk 2]
```diff
@@ -12,6 +12,28 @@ The clustering of microservices built to add functionality to an Entando applica
...
+
+You can use two different storage classes for your clustered vs non-clustered storage if your default class doesn't support `ReadWriteMany`. To do this add the following properties to your config map for the operator in the helm templates:
+
+```
+entando.k8s.operator.default.clustered.storage.class: "[your clustered RWX storage class]"
+entando.k8s.operator.default.non.clustered.storage.class: "[your RWO storage class]"
+```
+
+Set the values of both to the appropriate storage class for your configuration
```
Comment (avdev4j) [inline vuepress/docs/v6.3.2/docs/reference/caching-and-clustering.md:28]:
```suggestion
Set the values of both to the appropriate storage class for your configuration.
```

**Review by avdev4j — CHANGES_REQUESTED** [review]

**Comment by nshaw** [comment]
@joewhite101 , do the next files also need to be updated for these changes? I only see 6.3.2 and 6.3 here.

File: vuepress/docs/v6.3.2/docs/reference/caching-and-clustering.md (hunk 1 above)
Comment (joewhite101) [reply vuepress/docs/v6.3.2/docs/reference/caching-and-clustering.md:19]: Fixed.

**Review by avdev4j — APPROVED** [review]

**Pull request merged**